clients/privacy-center/components/modals/consent-request-modal/ConsentRequestModal.tsx

```diff
@@ -91,7 +86,6 @@ export const ConsentRequestModal: React.FC<ConsentRequestModalProps> = ({
       <VerificationForm
         isOpen={isOpen}
         onClose={onClose}
-        setAlert={setAlert}
```

Comment:
ahhh I love that you were able to clean this up 💯

Reply:
It felt good to remove all that code!